sim/deathknight/heart_strike.go

```diff
@@ -56,39 +66,15 @@ func (dk *Deathknight) newHeartStrikeSpell(isMainTarget bool, isDrw bool) *RuneS
 			}
 		},
 	}
-	if isDrw {
-		conf.DamageMultiplier *= .5
-		conf.Flags |= core.SpellFlagIgnoreAttackerModifiers
-	}
```

Comment:
This seems to have gone and we still need to apply those things to both main and off target DRW spells

Reply:
Fixed this, nice catch!